src/functionalTest/java/uk/gov/hmcts/reform/juddata/cameltest/JrdBatchApplicationTest.java

```diff
@@ -89,7 +88,7 @@ void testTasklet() throws Exception {
     @Test
     void testTaskletIdempotent() throws Exception {
         //clean context
-        SpringStarter.getInstance().restart();
+        //SpringStarter.getInstance().restart();
```

Comment:
@manukundloo-hmcts are we keeping this line commented?

Reply:
Thanks @ishratt i have removed the piece of code and pushed again.